scripts/db_measurements/measure_db.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
```

Comment:
I didn't check this script

Reply:
Will delete from this version anyways, it's just a place where Anton had stored his findings, it exists in the main repo.